### PR TITLE
Git guidelines

### PR DESCRIPTION
**Pull request by paked**
Here are my "Git Contribution Guidelines". I'd appreciate it if this was reviewed :)

cc: @zachlatta @MaxWofford 

Also, I have no idea why @jonleung's commit is showing up...

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -48,6 +48,88 @@ File names should follow the following guidelines:
...
+- This commit message does not give any context
+
+> Add the spark reactor source code into the document where we add our thoughts
+
+- This commit message is too long! It should be less than 50 characters!
+- It uses too many words and could be simplified
+
+If your PR does not satisfy these requirements a core member will point out your
+errors. If it is left inactive for more than 3 days, it will modified and merged
+for you.
```
Comment (zachlatta) [inline CONTRIBUTING.md]: I'm hesitant about this clause. What if we remove lines 129 - 131 entirely?

**Comment by zachlatta** [comment]
Looks good for the most part. I just left one comment.

I can handle removing @jonleung's commit.

**Comment by paked** [comment]
@zachlatta I'm trying to get better at git, do you reckon you could explain how I could do it? Also perhaps why it went wrong. That would be really appreciated :)

File: CONTRIBUTING.md (hunk 1 above)
Comment (paked) [inline CONTRIBUTING.md]: Sure. This works for me. Probably was premature optimisation anyway

**Comment by paked** [comment]
Right. So it looks like I accidentally reused @jonleung's branch lol. Whoops.

**Comment by paked** [comment]
@zachlatta What has to happen to get this merged?

**Comment by MaxWofford** [comment]
@paked This PR looks great! I've just moved it over here: #366

**Pull request closed without merging**